src/types/multiCheckbox.js

```diff
@@ -71,8 +71,10 @@ export default ngModule => {
           checkValidity(true);
         }
 
-        if (opts.expressionProperties && opts.expressionProperties.required) {
-          $scope.$watch($scope.options.expressionProperties.required, function(newValue) {
```

Comment:
Doing this wouldn't allow you to use `$viewValue` or `$modelValue` as part of your expression. So I just let angular-formly do it's magic to evaluate the expression property, and then we watch the result. I think that should work fine. Any problems with that?